jobio-back/backend.js

```diff
@@ -0,0 +1,126 @@
+const { formatDate } = require('./utils/functions');
+const { custom_queries } = require('./utils/queries');
+
+const { pool } = require('./db/db');
+const express = require('express');
+const cors = require('cors');
+
+const app = express();
+const PORT = 3001;
+
+app.use(express.json());
+app.use(cors({ origin: "*" }));
+
+app.listen(PORT, () => {
+    console.log(`the server is listening to the port ${PORT}`)
+})
+
+pool.query('SELECT NOW()', (err, res) => {
+    if (err) {
+        console.error('Error connecting to the database:', err);
+    } else {
+        console.log('Connected to the database on', res.rows[0].now);
+    }
+});
+
+app.get('/getOrgsNames', (req, res) => {
+    pool.query(custom_queries.getOrgNames)
+        .then(pooledData => { res.status(200).json(pooledData) })
```

Comment:
i recommend using async/await syntax instead of callbacks. it's more common today to use async/await.